src/commcare_cloud/ansible/roles/nginx/vars/cchq_ssl.yml

```diff
@@ -17,6 +17,17 @@ nginx_sites:
       path: "{{ www_home }}/hq_media/cache"
       max_size: "2g"
       inactive_time: "24h"
+   mappings:
+    - name: "$digest_submission"
+      variable: "$http_authorization"
+      value: "$server_name"
+   limit_req_zones:
+    - name: "restore"
+      zone: "$digest_submission"
+      rate: "6r/s"
+    - name: "submissions"
+      zone: "$digest_submission"
```

Comment:
I don't think we should use that zone as it's global. We want to restrict it per IP right? In that case we should use $binary_remote_addr